packages/faust-nx/src/hooks/usePreviewQuery.ts

```diff
@@ -0,0 +1,72 @@
+// eslint-disable-next-line import/extensions
+import {useState, useEffect} from 'react'
+import { ApolloClient, InMemoryCache, ApolloQueryResult, ApolloError } from '@apollo/client';
+import type { Node } from '@faustjs/react';
+import type { DocumentNode } from 'graphql';
+import isNil from 'lodash/isNil.js';
```

Comment:
Some of those imports are not used. They should be removed
<img width="997" alt="Screenshot 2022-08-17 at 13 59 40" src="https://user-images.githubusercontent.com/328805/185140365-f26943e1-69cd-449a-86d1-789ec37b49e4.png">
: